roles/ovirt-cluster-upgrade/library/ovirt_hosts.py

```diff
@@ -0,0 +1,464 @@
+#!/usr/bin/python
```

Comment:
Let's make a note somewhere, that we should remove this module from the role once the module update is backported to 2.3.1 and bump dependency to Ansible 2.3.1